pandas/core/indexing.py

```diff
@@ -88,7 +94,7 @@ class _IndexSlice:
            B1   10   11
     """
 
-    def __getitem__(self, arg):
+    def __getitem__(self, arg: Any) -> Any:
```

Comment:
Do these at least not have to be Hashable and/or some type of Mapping / Sequence?

Reply:
same.

Not prepared to justify each instance of Any. so will just remove them to streamline the process since that seems to be your preference.

Reply:
isn’t it possible just reveal type these then keep adding to a Union?

Reply:
There's probably not so many calls to __getitem__ from within the codebase as from the tests. but, yes, that seems a reasonable approach.